src/adapters/scene_direction.js

```diff
@@ -151,24 +153,41 @@ export default class SceneDirection {
   }
 
   showPolygon(route, vehicle) {
-    const geojson = {
-      "id": `route_${route.id}`,
-      "type": "line",
-      "source": `source_${route.id}`,
-      "layout": {
-        "line-join": "round",
-        "line-cap": "round",
-        "visibility": "visible"
-      },
-      "paint": {
-        "line-color": ["case",
-          ["boolean", ["feature-state", "isActive"], route.isActive],
-          MAIN_ROUTE_COLOR,
-          ALTERNATE_ROUTE_COLOR
-        ],
-        "line-width": 7
+    const geojson = (vehicle === "walking") ? {
+        "id": `route_${route.id}`,
+        "type": "symbol",
+        "source": `source_${route.id}`,
+        "layout": {
+          'icon-image': [
+            "case",
+            route.isActive,
+            'walking_bullet_active',
+            'walking_bullet_inactive',
+          ],
```

Comment:
this `case` is not necessary as the value is static.  I think it would be clearer to simply use:

```js
"icon-image": route.isActive ? "walking_bullet_active" :  "walking_bullet_inactive"
```